### ISSUE TITLE
AMS: Uncaught exception 'Exception' with message 'error in updating'

### ISSUE THREAD

**Issue by ryzom-pipeline** [ryzom/ryzomcore#181]
**[Original report](https://bitbucket.org/ryzom/ryzomcore/issue/181) by Jan Boon (Bitbucket: [Jan Boon](https://bitbucket.org/Jan Boon), ).**

----------------------------------------

When a user changing his own password

```
#!


Fatal error: Uncaught exception 'Exception' with message 'error in updating' in /srv/data/web/vhosts/*/private_php/ams/autoload/dblayer.php:213 Stack trace: #0 /srv/data/web/vhosts/*/private_php/ams/autoload/users.php(430): DBLayer->update('user', Array, 'Login = *') #1 /srv/data/web/vhosts/*/htdocs/ams/autoload/webusers.php(236): Users::setAmsPassword('*', '*') #2 /srv/data/web/vhosts/*/htdocs/ams/func/change_password.php(39): WebUsers::setPassword('*', '*') #3 /srv/data/web/vhosts/*/htdocs/ams/index.php(85): change_password() #4 {main} thrown in /srv/data/web/vhosts/*/private_php/ams/autoload/dblayer.php on line 213
```

**Comment by ryzom-pipeline** [comment]
**Original comment by Jan Boon (Bitbucket: [Jan Boon](https://bitbucket.org/Jan Boon), ).**

----------------------------------------

Similar error when changing e-mail address.

**Comment by ryzom-pipeline** [comment]
**Original comment by Matthew Lagoe (Bitbucket: [botanic](https://bitbucket.org/botanic), GitHub: [botanic](https://github.com/botanic)).**

----------------------------------------

Fix #181
